Tasks/Gradle/CodeAnalysis/SonarQube/common.ts

```diff
@@ -40,9 +41,15 @@ export function applySonarQubeParameters(toolRunner: ToolRunner): ToolRunner {
  * @param sqBuildFolder
  * @returns {Promise<void>} Promise resolved when all SQ integration actions are complete.
  */
-export function processSonarQubeIntegration(sqBuildFolder:string):Q.Promise<void> {
-    var sqRunSettings:SonarQubeRunSettings = getSonarQubeRunSettings(sqBuildFolder);
-    var sqMetrics:SonarQubeMetrics = getSonarQubeMetrics(sqRunSettings);
+export function processSonarQubeIntegration(sqTaskReportGlob: string): Q.Promise<void> {
```

Comment:
The main entry point into the post processing we do around SQ changes to accept a glob path. Note that is code is common to both Gradle and Maven and it is build agnostic.